SPECS/edk2/edk2.spec

```diff
@@ -330,6 +347,11 @@ tar -xf %{SOURCE3} --strip-components=1 --directory ArmPkg/Library/ArmSoftFloatL
 tar -xf %{SOURCE4} --strip-components=1 --wildcards "*/Drivers" "*/Features" "*/Platform" "*/Silicon"
 mkdir -p RedfishPkg/Library/JsonLib/jansson
 tar -xf %{SOURCE5} --strip-components=1 --directory RedfishPkg/Library/JsonLib/jansson
+
+# extract hvloader source into place
```

Comment:
Does it make sense to move this a bit down so that all hvloader sub-package steps are executed more at the bottom of each '%' section and not 'in between'?

Reply:
Not opposed, will make that tweak.